src/App.tsx

```diff
@@ -1,13 +1,62 @@
 import React from 'react';
 import { Route, Routes } from 'react-router-dom';
-import { LoginViews } from './views/LoginViews';
-
-import './_App.scss';
+import { LoginView } from './views/App/LoginView/LoginView';
+import { AllStudentsView } from './views/App/AllStudentsView/AllStudentsView';
+import { StudentView } from './views/App/StudentView/StudentView';
+import { AdminLoginView } from './views/Admin/AdminLoginView/AdminLoginView';
+import { AdminImportView } from './views/Admin/AdminImportView/AdminImportView';
+import { AdminListView } from './views/Admin/AdminListView/AdminListView';
+import { ShowCvView } from './views/App/ShowCvView/ShowCvView';
+import { EditCvView } from './views/App/EditCvView/EditCvView';
+import { StudentRegisterView } from './views/Register/StudentRegisterView/StudentRegisterView';
+import { InterviewView } from './views/App/InterviewView/InterviewView';
+import { HrRegisterView } from './views/Register/HrRegisterView/HrRegisterView';
+//import { ErrorView } from './views/ErrorView';
+//import { useDispatch, useSelector } from 'react-redux';
+//import { RootState } from './redux/store';
 
 export const App = () => {
-  return (
-    <Routes>
-      <Route path="/" element={<LoginViews />} />
-    </Routes>
-  );
+  // global variable redux toolkit
+  //const { token, type } = useSelector((state: RootState) => state.token);
+  //const dispatch = useDispatch();
+
+  switch (window.location.host.split('.')[0]) {
```

Comment:
Ojojoj, bardzo nie po React-Routerowemu. Poszukajcie jak to się powinno tak robić. Inaczej mogą być problemy i nie wygląda to ładnie